Client/Frontend/Widgets/CellWithRoundedButton.swift

```diff
@@ -57,17 +57,19 @@ class CellWithRoundedButton: UITableViewCell, NotificationThemeable, ReusableCel
         self.selectionStyle = .default
 
         contentView.addSubview(roundedButton)
-        roundedButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: 30, bottom: 0, right: 0)
+
+        roundedButton.setInsets(forContentPadding: UIEdgeInsets(top: 14, left: 14, bottom: 14, right: 14),
+                                imageTitlePadding: 11)
```

Comment:
nit: we could put some of the UX constants inside the CellWithRoundedButtonUX. We also started moving UX stuct inside the related classes directly (instead of them being global).